spec/0.0.2/examples/template/Nulecule

```diff
@@ -15,10 +15,11 @@ graph:
     source: "docker://registry.example.com/some/database"
   - artifacts: 
       provider1: 
-        - "file://artifacts/provider1/pod.json"
-        - "file://artifacts/provider1/service.json"
+        - source: "https://github.com/foo/bar/provider1.git"
+          path: /my/provider/files/
+          tag: release-1
       provider2: 
-        - "file://artifacts/provider2/file.json"
+        - "file://artifacts/provider2/"
```

Comment:
Per IRC discussion the artifacts are a list of either a URL string or a source control repository object. I agree it's a bit odd to mix string and key:val objects but it works. 'source' is redundant.